### PR TITLE
Adapt to new release workflow

### PR DESCRIPTION
**Pull request by mschwoer**

### CODE REVIEW

File: .github/workflows/create_release.yml [hunk 1]
```diff
@@ -1,200 +1,22 @@
...
-      commit_to_release:
-        description: 'Enter commit hash to release (example: ef4037cb571f99cb4919b520fde7174972aae473)'
-        required: true
-      tag_to_release:
-        description: 'Enter tag to release (example: v1.5.5)'
-        required: true
-
-
-name: Create Draft Release
+      commitish_to_release:
```
Comment (GeorgWa) [inline .github/workflows/create_release.yml:7]: is this a typo? :D

**Review by GeorgWa — APPROVED** [review]
LGTM
Should we remove the old files soon?

File: release/macos/distribution.xml [hunk 2]
```diff
@@ -1,17 +1,17 @@
 <?xml version="1.0" encoding="utf-8" standalone="no"?>
 <installer-script minSpecVersion="1.000000">
-    <title>alphaDIA 1.7.2</title>
+    <title>AlphaDIA 1.7.2</title>
```
Comment (GeorgWa) [inline release/macos/distribution.xml:3]: have the changes in the `distribution.xml` and the `info.plist` been tested?

**Comment by GeorgWa** [comment]
I just tested the bundles shipped with https://github.com/MannLabs/alphadia/releases/tag/untagged-8e2ba4d90dd8c50c45d2
And can't launch the installed version. Is this the right installer to test?

File: release/macos/distribution.xml (hunk 2 above)
Comment (mschwoer) [reply release/macos/distribution.xml:3]: yup I tested all installers

**Pull request merged**